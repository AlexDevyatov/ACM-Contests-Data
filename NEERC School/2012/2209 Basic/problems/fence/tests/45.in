13
-317 184
-297 -214
-268 -263
-160 -316
-90 -340
260 -332
335 -321
338 -319
325 47
313 288
-5 330
-212 339
-314 225
329 -240
