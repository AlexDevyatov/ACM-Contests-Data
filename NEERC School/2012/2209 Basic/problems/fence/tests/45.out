14
-317 184
-297 -214
-268 -263
-160 -316
-90 -340
260 -332
335 -321
329 -240
338 -319
325 47
313 288
-5 330
-212 339
-314 225
